setup-e2e-tests.ps1

```diff
@@ -98,7 +104,7 @@ if (Test-Path $output)
   Remove-Item $output -Recurse -Force -ErrorAction Ignore
 }
 
-$AdditionalPackArgs = @("-c", "Release", "-p:FunctionsRuntimeVersion=$FunctionsRuntimeVersion")
+$AdditionalPackArgs = @("-c", "Release", "-p:FunctionsRuntimeVersion=$FunctionsRuntimeVersion", "-p:DotnetVersion=$DotnetVersion")
```

Comment:
Can you remind me who sets this environment variable when we are triggering this pipeline for a PR build? We need to run it with both values rite? So does this gets called twice with different values?

Reply:
Thank you for reminding me this question. @jviau do you know if the ask is to run tests with both net7 and net48 every time or is it either-or based on pipeline configuration? If not, I will double check with Fabio and update here.

Reply:
That depends on how our infra will handle it. _Ideally_, they both would run in the same pipeline together. However, I know we had issues with netfx tests before on the linux job, so we may need to address that.

Reply:
Update ci.yml to run both everytime.